pkg/resources/csi/kubevirt/rbac.go

```diff
@@ -170,7 +170,7 @@ func ControllerRoleBindingReconciler(c *kubermaticv1.Cluster) reconciling.NamedR
 			r.RoleRef = rbacv1.RoleRef{
 				APIGroup: "rbac.authorization.k8s.io",
 				Kind:     "ClusterRole",
-				Name:     resources.KubeVirtCSIControllerName,
+				Name:     resources.KubeVirtCSIClusterRoleName,
```

Comment:
The other change(adding `get` verb) makes sense. However, this change, at least to me, seems like we have a bigger issue with the RBAC since this role/rolebinding is being created in the seed cluster which is not required IMO. But for now it's good to go from my side.

Reply:
I agree with @ahmedwaleedmalik here. This change doesn't make quite sense to me and the whole RBAC model must be completely re-worked, by the end of the day the CSI controller is working on . However this should be in another PR.as the KubeVirt CSi Driver is only looking at Tenant cluster and Infra cluster not the seed. And this PR applies things on seed.

Reply:
> Till we get the RBAC model reworked, doesn't it make sense to have these changes in?

Not really as this might even complicate things more when this will be refactored because this change will then point to a real cluster role that will introduce confusion to whom is gonna work on it. 

> Without the binding update it just keeps complaining that the cluster role doesn't exist group "kubevirt.io" in the namespace "cluster-rl877lc7c6": RBAC: clusterrole.rbac.authorization.k8s.io "csi-controller" not found.

That means you would like to get rid of a log message message and exchange it with a logical error that will go silent for god knows when. IMHO, I would prefer to leave this error message to mark down this issue for the future instead of hiding somewhere in the code.

Reply:
@moadqassem  I have reverted the change in question.